alibi_detect/utils/fetching.py

```diff
@@ -514,7 +514,7 @@ def fetch_detector(filepath: Union[str, os.PathLike],
         if model_type == 'weights':
             kwargs = get_pixelcnn_default_kwargs()
     detector = load_detector(filepath, **kwargs)
-    return detector
+    return detector  # type: ignore[return-value] # Since load_detector returns drift detectors but "Data" doesn't inc.
```

Comment:
This ignore can be removed once all detectors use the config based approach.

Reply:
Can you add this comment in the source code with a `TODO` please?

Also `but "Data" doesn't inc.`...?